gemfire/spring-cloud-starter-stream-sink-gemfire/src/test/java/org/springframework/cloud/stream/app/gemfire/sink/GemfireSinkIntegrationTests.java

```diff
@@ -37,24 +48,16 @@
 import org.springframework.messaging.support.GenericMessage;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
-import javax.annotation.Resource;
-import java.io.File;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.concurrent.TimeUnit;
-
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertTrue;
+import com.gemstone.gemfire.cache.Region;
 
 
 /**
  * @author David Turanski
  */
 @RunWith(SpringJUnit4ClassRunner.class)
 @SpringApplicationConfiguration(classes = {GemfireSinkIntegrationTests.GemfireSinkApplication.class})
-@IntegrationTest({"regionName=Stocks", "keyExpression='key'", "hostAddresses=localhost:42424", "connectType=server"})
+@IntegrationTest({"gemfire.region.regionName=Stocks", "gemfire.keyExpression='key'", "gemfire.pool.hostAddresses=localhost:42424",
+		"gemfire.pool.connectType=server"})
 @EnableConfigurationProperties(GemfireSinkProperties.class)
 public class GemfireSinkIntegrationTests {
```

Comment:
This test works for me, but I don't like that explicit `port` for embedded Gemfire.
Although looks like we can't have a random one there...

Anyway looks like that https://github.com/spring-cloud/spring-cloud-stream-modules/issues/49 is most like some race condition during connection. Or in Spring Data Gemfire or in the Gemfire per se.

After upgrading to SDG-1.8.1.RELEASE and, therefore, Gemfire 8.2.0 in SI, looks like we have stopped to fail in our CI plans for similar reason. 

Not sure if we should upgrade here to SD-Hopper right now, but at least we should keep that in mind for the future, e.g. after switching to Boot-1.4.